tests/bluetooth/bsim_bt/edtt_ble_test_app/hci_test_app/prj_legacy.conf

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_BT=y
+CONFIG_BT_HCI_RAW=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_CENTRAL=y
+CONFIG_BT_DEBUG_LOG=y
+CONFIG_BT_ECC=y
+CONFIG_BT_TINYCRYPT_ECC=y
+
+##
+## Enabling BT_CTRL_DTM_HCI requires BT_LL_SW_SPLIT which requires BT_CTRL
+##
+CONFIG_BT_CTLR=y
+
+CONFIG_BT_LL_SW_SPLIT=y
+CONFIG_BT_CTLR_CRYPTO=y
+CONFIG_BT_CTLR_LE_ENC=y
+CONFIG_BT_CTLR_PRIVACY=y
+CONFIG_BT_CTLR_FILTER=y
+CONFIG_BT_CTLR_DTM_HCI=y
+CONFIG_BT_CTLR_TX_BUFFER_SIZE=60
+CONFIG_BT_CTLR_DATA_LENGTH_MAX=60
+
+CONFIG_SYS_POWER_MANAGEMENT=y
+CONFIG_TICKLESS_IDLE=y
+CONFIG_TICKLESS_KERNEL=y
```

Comment:
At some point in the future the default LL will switch to the refactored LLCP. Wouldn't it be better to explicitly select the legacy LLCP here:

`# Disabling BT_LL_SW_SPLIT_LLCP_LEGACY requires BT_CTLR_ADVANCED_FEATURES`
`CONFIG_BT_CTLR_ADVANCED_FEATURES=y`
`CONFIG_BT_LL_SW_SPLIT_LLCP_LEGACY=y`

Even though legacy is default now, when changing the default it is easy to overlook that this file needs to be changed at that time.

Reply:
ACK